tests/test_management.py

```diff
@@ -120,47 +149,80 @@ def test_transmit_course_metadata_task_with_error(
             catalog=1,
             name='Dummy Enterprise',
         )
-        enterprise_uuid_for_success = str(dummy_enterprise_customer.uuid)
+        factories.DegreedEnterpriseCustomerConfigurationFactory(
+            enterprise_customer=dummy_enterprise_customer,
+            key='key',
+            secret='secret',
+            degreed_company_id='Degreed Company',
+            active=True,
+        )
         factories.SAPSuccessFactorsEnterpriseCustomerConfigurationFactory(
             enterprise_customer=dummy_enterprise_customer,
             sapsf_base_url='http://enterprise.successfactors.com/',
             key='key',
             secret='secret',
             active=True,
         )
+
+        enterprise_uuid_for_success = str(dummy_enterprise_customer.uuid)
         self.mock_ent_courses_api_with_pagination(
             enterprise_uuid=enterprise_uuid_for_success,
             course_run_ids=[course_run_id_for_success]
         )
 
-        expected_dump = (
+        degreed_expected_dump = (
+            '{"courses": [{"authors": [], "categoryTags": [], "contentId": "'+course_run_id_for_success+'", '
+            '"costType": "Paid", "description": "edX Demonstration Course", "difficulty": "", "duration": 0, '
+            '"format": "Instructor", "imageUrl": "", "institution": "", "language": "en", '
+            '"publishDate": "2013-02-05", "title": "edX Demonstration Course", '
+            '"url": "'+settings.LMS_ROOT_URL+'/enterprise/'+enterprise_uuid_for_success+'/course/'
+            ''+course_run_id_for_success+'/enroll/", "videoUrl": ""}], '
+            '"orgCode": "Degreed Company", "providerCode": "DEGREED"}'
+        )
```

Comment:
@UmanShahzad Not sure if `pylint` would complain, but putting spaces around `+` would improve readability here, I think.

Reply:
It won't, so yes, sure :) :+1: